### ISSUE TITLE
LBaaS documentation

### ISSUE THREAD

**Issue by gbraad** [redhat-openstack/website#404]
I would like to modify the `source/networking/lbaas.md` documentation as it is currently out-of-date and refers to a specialized rhel-http image to setup the environment.

Although I have written a multi-tier setup, which also uses a load balancer, but the instructions are not dependent of the RHEL images as plain cirros is utilized.

https://github.com/gbraad/openstack-handsonlabs/blob/master/neutron/building-multitier-application.md

What do you prefer, inclusion / merging or just updating the current documentation so that it will works as-is (bu changing the instructions to use `os-neutron-lbaas-installl=y`) ?

**Comment by gbraad** [comment]
How about NetworkManager and network service?

In my guide I do:

```
$ chkconfig NetworkManager off
$ chkconfig network on
$ systemctl stop NetworkManager.service
$ systemctl start network.service
```

to disable NetworkManager.

**Comment by hguemar** [comment]
@gbraad just send a patch but we only need the systemctl bits on EL7, and I'll get it merged

**Comment by gbraad** [comment]
#564

```
$ systemctl disable NetworkManager
$ systemctl enable network
$ systemctl stop NetworkManager.service
$ systemctl start network.service
```

**Comment by hguemar** [comment]
Thank you @gbraad :+1: 
Closed as PR #564 got merged
